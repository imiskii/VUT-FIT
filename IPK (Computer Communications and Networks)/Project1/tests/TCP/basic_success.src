HELLO
SOLVE (+ 5 7)
SOLVE (- 2 10)
SOLVE (* 8 3)
SOLVE (* 0 2)
SOLVE (/ 64 8)
BYE