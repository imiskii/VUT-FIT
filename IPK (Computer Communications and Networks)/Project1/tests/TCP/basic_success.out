HELLO
RESULT 12
RESULT -8
RESULT 24
RESULT 0
RESULT 8
BYE

